app/src/mainwindow2.cpp

```diff
@@ -1295,6 +1297,54 @@ void MainWindow2::importPalette()
     }
 }
 
+void MainWindow2::savePalette()
+{
+    FileDialog FileDialog(this);
+    QString filePath = FileDialog.saveFile(FileType::PALETTE_XML);
+    if (!filePath.isEmpty())
+    {
+        mEditor->object()->exportPalette(filePath);
+    }
+}
+
+void MainWindow2::openPalette()
+{
+    int count = 0;
+    int maxNumber = mEditor->object()->getColourCount();
+    bool openPalet = true;
```

Comment:
typo, should be openPalette. It's a minor thing but makes it much easier to search.